.pre-commit-config.yaml

```diff
@@ -1,14 +1,13 @@
 ci:
   autofix_prs: false
-
+exclude: "{{ cookiecutter.project_slug }}"
```

Comment:
This is probably better than specifying everywhere. We have the linting of the template CI anyway.